kalite/coachreports/static/js/coachreports/coach_reports/router.js

```diff
@@ -0,0 +1,39 @@
+CoachReportRouter = Backbone.Router.extend({
+    initialize: function(options) {
+        _.bindAll(this);
+        this.state_model = new Backbone.Model();
+        this.listenTo(this.state_model, "change", this.set_url);
+    },
+
+    routes: {
+        "(:facility/)(:group/)(:report/)":    "report"
```

Comment:
What's the intent here? The documentation suggests to me that, if `:facility/` is absent, then a url like `somegroup/somereport/` will result in `facility == "somegroup"` and `group == "somereport"`.

Can optional parameters be nested? If so I think `(:facility(/:group(/:report/)))` might be what's intended. If not, you can use [regular expressions in routes](http://backbonejs.org/#Router-route) to achieve the same thing.